index.html

```diff
@@ -4176,12 +4205,29 @@ <h5>Existing implementations</h5>
 <dt>leaflet-api</dt>
 <dd><a>full support</a>:
 <!-- details about what is/isn't supported -->
+<p>
+These implementations all provide straightforward APIs for adding markers to locations using a default icon.
+</p>
 </dd><dt>open-layers-api</dt>
 <dd><a>supported, with limitations</a>:
 <!-- details about what is/isn't supported -->
+<p>
+Rather than providing a straightforward API, OpenLayers requires a number of steps to be followed:
+providing icon imagery (as there is no default imagery provided),
+creating icon features, creating a map layer to hold thos features, and finally ading the layer to a map.
```

Comment:
> creating a map layer to hold **thos** features, and finally **ading** the layer to a map.

Reply:
The perils of writing English text in a code editor and not running a spellcheck over it :-)